eth/vm/base.py

```diff
@@ -668,7 +668,13 @@ def validate_uncle(cls,
                 f"Uncle's gas usage ({uncle.gas_used}) is above "
                 f"the limit ({uncle.gas_limit})"
             )
-        validate_gas_limit(uncle.gas_limit, uncle_parent.gas_limit)
+
+        uncle_parent_gas_limit = uncle_parent.gas_limit
+        if not hasattr(uncle_parent, 'base_fee_per_gas') and hasattr(uncle, 'base_fee_per_gas'):
+            # if Berlin -> London transition, double the parent limit for validation
+            uncle_parent_gas_limit *= 2
+
+        validate_gas_limit(uncle.gas_limit, uncle_parent_gas_limit)
```

Comment:
Oof, this is a bummer, but I don't have a better suggestion right now.

Reply:
Hah, yes, same... I think it can use a refactor at some point for sure.